proto/Enums/SocialAction.proto

```diff
@@ -22,5 +22,5 @@ enum SocialAction {
 	UNREGISTER_PUSH_NOTIFICATION_SOCIAL_ACTION = 10102;
 	UPDATE_NOTIFICATION = 10103;
 	OPT_OUT_PUSH_NOTIFICATION_CATEGORY_SOCIAL_ACTION = 10104;
-	GET_INBOX = 10105;
+	GET_INBOX_SOCIAL_ACTION = 10105;
```

Comment:
Nope, it's GET_INBOX. It was correct.

Reply:
ok revert 👍

Reply:
https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/535789e43539e79403083fef746a7292f18ffce2